qa/tasks/cephfs/test_fscrypt.py

```diff
@@ -47,3 +54,31 @@ def test_fscrypt(self):
         # skipped for now because of not supporting features in kernel or kceph.
         self.assertEqual(proc.returncode, 0)
         self.assertIn('Passed all 26 tests', stdout)
+
+    def test_fscrypt_dummy_encryption_with_quick_group(self):
+        if not check_mounter():
+            self.skipTest('Requires kernel client; xfstests-dev not '\
+                          'compatible with ceph-fuse ATM.')
+
+        self.write_local_config('test_dummy_encryption')
+
+        # XXX: check_status is set to False so that we can check for command's
+        # failure on our own (since this command doesn't set right error code
+        # and error message in some cases) and print custom log messages
+        # accordingly. This will take a long time and set the timeout to 5 hours.
+        proc = self.mount_a.client_remote.run(args=['sudo', './check',
+            '-g', 'quick', '-E', './ceph.exclude'], cwd=self.xfstests_repo_path,
```

Comment:
For convenience you can also pass command arguments as a single string instead as a list of string -
```suggestion
        proc = self.mount_a.client_remote.run(args='sudo ./check -g quick -E ./ceph.exclude', cwd=self.xfstests_repo_path,
```

Reply:
Yeah, I just followed the old code's style.

Reply:
It looks more tidy when breaking it into two lines. Let's keep the old style.